src/main/flight/pid.c

```diff
@@ -105,11 +108,13 @@ PG_RESET_TEMPLATE(pidConfig_t, pidConfig,
     .pid_process_denom = PID_PROCESS_DENOM_DEFAULT,
     .runaway_takeoff_prevention = true,
     .runaway_takeoff_deactivate_throttle = 20,  // throttle level % needed to accumulate deactivation time
-    .runaway_takeoff_deactivate_delay = 500     // Accumulated time (in milliseconds) before deactivation in successful takeoff
+    .runaway_takeoff_deactivate_delay = 500,    // Accumulated time (in milliseconds) before deactivation in successful takeoff
+    .pid_adjust_aux_channel = 0
```

Comment:
Please add commas after the last line of these lists - this will make for cleaner diffs after the next addition.

Reply:
A comma after ".pid_adjust_aux_channel"?

Reply:
Ok, i read it in the guidelines. Nevertheless, as long as the docs are English only, failures may happen.